### PR TITLE
Added Product Listing Page

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Product Listing Page

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
Clean and structured code, Good use of variables. It would be great if you attach a screenshot/video for a preview

File: html/luggage.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      href="https://concreteui.netlify.app/styles/index.css"
+    />
+    <link rel="stylesheet" href="/styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
Comment (Hamdrive) [inline html/luggage.html]: Consider importing the link directly into your main CSS, thus saving you the need to import it as link everytime in HTML

**Review by Hamdrive — COMMENTED** [review]
Great work done, will suggest to also add a screenshot or short GIF demonstrating the website, inaddition to the deploy link

Have made a comment about importing FontAwesome into CSS instead of HTML, do look into it.

File: html/luggage.html (hunk 1 above)
Comment (nareshpoonia) [reply html/luggage.html]: Thanks for pointing that out, I have removed the font awesome link from all html files & put it in main CSS instead

**Pull request merged**